easybuild/easyconfigs/b/BayesAss/BayesAss-3.0.4-foss-2016a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'BayesAss'
+version = '3.0.3'
```

Comment:
this doesn't match with the filename, should it be 3.0.4 (if not, please fix the filename to use 3.0.3)